models/solana_utils/solana_utils_latest_balances.sql

```diff
@@ -2,6 +2,7 @@
   config(
         alias = alias('latest_balances'),
```

Comment:
Can you add a tag here like  `schema = 'solana_utils'`? It should help with the CI